monocle/github/graphql.py

```diff
@@ -86,9 +90,12 @@ def getRateLimit(self):
         return data['data']['rateLimit']
 
     @retry(
-        before=before_log(log, logging.INFO), wait=wait_exponential(min=10, max=300),
+        after=after_log(log, logging.INFO),
+        wait=wait_fixed(1),
```

Comment:
What about using retry_unless_exception_type('Timeout')  because if we have a HTTP timeout or other transport/HTTP errors we might need to retry and even with wait_exponential. We don't want to retry the same query when we have that API timeout we let the caller rework the size.